dd-sdk-android-bridge/src/main/kotlin/com/datadog/android/bridge/internal/BridgeRum.kt

```diff
@@ -163,4 +160,8 @@ internal class BridgeRum : DdRum {
     }
 
     // endregion
+
+    companion object {
+        internal const val TIMESTAMP = "_dd.timestamp"
```

Comment:
The way I remember you remove this at some point from the attributes, this means you will have a duplicated key in the main module right ?